charts/incubator/handbrake/values.yaml

```
@@ -0,0 +1,51 @@
+#
+# IMPORTANT NOTE
+#
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
+
+image:
+  # -- image repository
+  repository: jlesage/handbrake
+  # -- image tag
+  # @default -- chart.appVersion
+  tag: v1.24.2
```

```suggestion
  tag:
```
We recoomend having this blank - the tag above it 'chart.appVersion' picks up the apperversion in Chart.yaml.  Saves maintaining in two spots